tests/test_peak_detection.py

```diff
@@ -1256,11 +1260,11 @@ def test_noisy_data_A1(noisy_data_A1):
     expected_peak_indices = [
         19,
         580,
-        1165,
-        1728,
+        1166,
```

Comment:
[note for future] in general going forward, we're going to need a way to lock in a particular filter set with old data.  we don't want to be constantly changing these numbers just because we decided that a 31 Hz Bessel cutoff works better than 30 Hz